2.4.0/spec/technical_info.md

```diff
@@ -120,10 +120,13 @@ Content-Type: application/json
 ]
 ```
 
-| Parameter | Type        | Description                                     |
-|-----------|-------------|-------------------------------------------------|
-| mac       | string (17) | Mac address of device                           |
-| serviceType   | string      | Which network service the mac originates from   |
+| Parameter     | Type              | Description                                           |
+|---------------|-------------------|-------------------------------------------------------|
+| mac           | string (17)       | Mac address of device                                 |
+| serviceType   | string            | Which network service the mac originates from         |
+| vlan          | integer           | Some services can exist on multiple vlans             |
+| port          | string            | Distinguish port, In case uplink mac's are included   |
+| learntAt      | string (iso8601)  | Some switches keep age of mac table entry             |
```

Comment:
```suggestion
| Parameter    | Type             | Description                                                   |
|--------------|------------------|---------------------------------------------------------------|
| mac          | string (17)      | Mac address of device                                         |
| serviceType  | string           | Which network service the mac originates from                 |
| vlan         | integer          | Some services can exist on multiple vlans                     |
| port         | string           | Distinguish port, In case uplink mac addresses are included   |
| learntAt     | string (iso8601) | Some switches keep age of mac address table entry             |
```